tests/integration/targets/ec2_asg/roles/ec2_asg/tasks/env_setup.yml

```diff
@@ -1,5 +1,5 @@
 - name: Run ec2_asg integration tests.
-
+  run_once: '{{ setup_run_once | default("no") | bool }}'
```

Comment:
I would instead use `tests/integration/targets/ec2_asg/roles/ec2_asg/default/main.yaml` to define a default `setup_run_once` value for the role.

Reply:
Done. Thanks!